nmtwizard/preprocess/consumer.py

```diff
@@ -137,10 +141,25 @@ def _log_info(file_info, file_summary, proc, units):
             summary = self._summary[proc]
             if summary:
                 all_files_summary = summary.pop(None)
-                _log_info("for the whole sample ", all_files_summary, proc, units)
+                total_sentence_number = sum(
+                    [v.get("linesampled") for v in self._sampler_summary.values()]
```

Comment:
Also, can the field "linesampled" be missing?
* If yes, it should be `v.get("linesampled", 0)`
* If not, it should be `v["linesampled"]`